Jenkinsfile

```diff
@@ -69,6 +69,7 @@ pipeline {
 				}
 			}
 			steps {
+				sh 'git submodule update --init --recursive'
```

Comment:
Not sure this will work but the intention is that we update the swagger file (by updating the submodule) on every build. Reference: https://stackoverflow.com/questions/42290133/jenkins-pipeline-git-command-submodule-update

cc @abesto